### ISSUE TITLE
Xiexie Toon Emission seems to have broke, no longer shows the set emissive colour

### ISSUE THREAD

**Issue by MahSandwich** [Yellow-Dog-Man/Resonite-Issues#652]
### Describe the bug?

I spawned an object into a world and noticed the emissive stripes were no longer emissive and so I began investigating.
I searched and found https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/237 but I think it is a different issue as this was working on the previous Resonite version. (Outline Emissive still works).

### To Reproduce

Create New XiexieToon Material and give a value for EmissionColor

### Expected behavior

EmissionColor in the world should match the EmissionColor in the editor.

### Screenshots

Compared with PBS_Metallic with the same AlbedoColor and EmissionColor
![Photo in Hikari Akimori World](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/70416315/172ccaf2-e129-4468-8e86-6934d8198be8)

### Resonite Version Number

Beta 2023.11.11.1231

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Valve Index, Desktop

### Log Files

[SYS0 - 2023.11.11.1231 - 2023-11-12 07_00_05.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13327912/SYS0.-.2023.11.11.1231.-.2023-11-12.07_00_05.log)

### Reporters

Hikari Akimori (Discord: hikari_akimori)

**Comment by lxw404** [comment]
This seems to be an issue with how the toon material is expecting an `EmissionMap` texture or else it defaults that texture to black, whereas other materials like PBS default this texture to white.  You can test this by putting a `SolidColorTexture` set to white into the toon material's `EmissionMap`.

**Comment by shiftyscales** [comment]
Yeah, I've also noticed in past that for some reason XiexeToon emissions are only active while they have an emission map specified. As covered in #1184, we will be replacing XiexeToon outright at some point, so this might not be fixed prior to then.

Seeking input from @Geenz - my guess it that the emission property block probably doesn't exist on the shader variants without an emission map for some reason. If this is a quick fix, it could be worthwhile- otherwise, I'd consider this low-priority because it can be worked around by using a low-res SolidColorTexture.
